google/cloud/testing_util/example_driver.cc

```diff
@@ -61,17 +61,21 @@ int Example::Run(int argc, char const* const argv[]) try {
 
   std::string const command_name = argv[1];
   auto command = commands_.find(command_name);
-  if (commands_.end() == command) {
-    PrintUsage(argv[0], "Unknown command: " + command_name);
-    return 1;
-  }
+    if (commands_.end() == command) {
```

Comment:
Fix indentation.

Are you able to run the "checkers" build to automatically fix these?

Reply:
I have fixed the indentation. How do I run "checkers" build, do we have any documentation for that?

Reply:
https://github.com/googleapis/google-cloud-cpp/tree/main/doc/contributor